pkg/sdn/plugin/node.go

```diff
@@ -63,6 +78,10 @@ func NewNodePlugin(pluginName string, osClient *osclient.Client, kClient *kclien
 		}
 	}
 
+	if err := writeCNIConfig(masterKubeConfig, hostname, mtu); err != nil {
```

Comment:
This still needs to be addressed; if we can't change it then please file a bug against CNI and link to it